tests/unit/backend/chalice/api_server/test_app.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+
+from tests.unit.backend.chalice.api_server import BaseAPITest
+
+
+class TestAPI(BaseAPITest, unittest.TestCase):
```

Comment:
Curious, how long does this take? This seems more like an end2end test to me which would be more appropriate in the functional tests directory.

Reply:
+1

Reply:
After setup, they run at millisecond. spinning up and populating the database takes longer.